kubernetes/kserve/kserve_wrapper/__main__.py

```diff
@@ -74,26 +74,33 @@ def parse_config():
         model_store = DEFAULT_MODEL_STORE
     logging.info(
         "Wrapper : Model names %s, inference address %s, management address %s, model store %s",
-        model_names, inference_address, management_address, model_store)
+        model_names,
+        inference_address,
+        management_address,
+        model_store,
+    )
 
     return model_names, inference_address, management_address, model_store
 
 
 if __name__ == "__main__":
 
-    model_names, inference_address, management_address, model_dir = parse_config(
-    )
+    model_names, inference_address, management_address, model_dir = parse_config()
 
     models = []
 
     for model_name in model_names:
 
-        model = TorchserveModel(model_name, inference_address,
-                                management_address, model_dir)
+        model = TorchserveModel(
+            model_name, inference_address, management_address, model_dir
+        )
+        model.load()
```

Comment:
@jagadeeshi2i  Please add a comment here mentioning why we are loading the model here